src/workerd/api/node/tests/tls-nodejs-test.js

```diff
@@ -282,3 +283,356 @@ export const tlsConnectGivenSocket = {
     await Promise.all(promises);
   },
 };
+
+export const testSecureContext = {
+  async test() {
+    throws(() => tls.connect({ port: 42, secureContext: {} }), {
+      code: 'ERR_TLS_INVALID_CONTEXT',
+    });
+
+    doesNotThrow(() => {
+      const secureContext = tls.createSecureContext({});
```

Comment:
Should have a test that also covers the `tls.SecureContext()` without new or using the `createSecureContext()` method so we make sure we don't regress.